Integration/macOSSwiftIntegration/main.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  main.swift
+//  macOSSwiftIntegration
+//
+//  Created by Bogdan Poplauschi on 18/10/2018.
+//  Copyright © 2018 CocoaLumberjack. All rights reserved.
+//
```

Comment:
Don't forget to copy the new copyright header in here, too.

Reply:
Yeap, good catch.

Reply:
@ffried Hmm. We should add a check into Danger, that way we don't get into this spot again.

Reply:
For new files, this shouldn’t happen again now. I’ve added the templates to the Xcode project files. so this should only be a temporary problem now.

Reply:
Yeah but people always remove templates :) same as for the GitHub issue template.

Here's what I come up with:
```ruby
# Check Copyright - first 7 lines only
expected_copyright_lines = [
"// Software License Agreement (BSD License)",
"//",
"// Copyright (c) 2010-2018, Deusty, LLC",
"// All rights reserved.",
"//",
"// Redistribution and use of this software in source and binary forms,",
"// with or without modification, are permitted provided that the following conditions are met:",
"//",
"// * Redistributions of source code must retain the above copyright notice,",
"//   this list of conditions and the following disclaimer.",
"//",
"// * Neither the name of Deusty nor the names of its contributors may be used",
"//   to endorse or promote products derived from this software without specific",
"//   prior written permission of Deusty, LLC."
]

sourcefiles_to_check = (git.modified_files + git.added_files).uniq
invalid_copyright = false
sourcefiles_to_check.each do |sourcefile|
  fileExtension = File.extname(sourcefile)
  next unless (fileExtension.include?(".swift") or fileExtension.include?(".h") or fileExtension.include?(".m"))
  next unless File.file?(sourcefile)
  File.readlines(sourcefile)[0..13].each_with_index do |line, index|
    if !line.include?(expected_copyright_lines[index])
      invalid_copyright = true
      markdown("Invalid copyright: " + sourcefile)
      break
    end
  end
end
if invalid_copyright === true
  warn("Copyright is not valid. See our default copyright in all of our files")
end
```

Reply:
:) I was initially checking for the default 7 lines Copyright, but then I thought we can actually check for the one we need.
I can also add a different Copyright for Demos.
Yeap, 13 is equal to the number of `expected_copyright_lines`. I thought I could check it directly like if the beginning of the file is x. Not a final version definitely.

Reply:
For Demos, we need to check the following pattern:

```
//
//  $FILENAME
//  $TARGET
//
//  CococaLumberjack Demos
//
```

While `$FILENAME` is easy (just the name of the file), `$TARGET` is a little more complicated. For now I'd just suggest we ignore it. Especially since there's not always a unique target, in which case `$TARGET` might be chosen differently. But if the rest matches, I'm fine with it.

Reply:
Check out #972 - I also included a simple check for the Demos.